Directory.Build.props

```
@@ -260,6 +264,7 @@
   <Import Project="eng\targets\Java.Common.props"  Condition="'$(MSBuildProjectExtension)' == '.javaproj'" />
   <Import Project="eng\targets\Helix.props" Condition=" $(IsTestProject) " />
   <Import Project="eng\targets\FunctionalTestWithAssets.props" Condition=" $(IsTestProject) " />
+  <Import Project="eng\testing\linker\trimmingTests.props" Condition="'$(IsPublishedAppTestProject)' == 'true'" />
```

```suggestion
  <Import Project="eng\testing\linker\trimmingTests.props" Condition="'$(IsPublishedAppTestProject)' == 'true'" />
  <Import Project="eng\targets\Helix.props" Condition=" $(IsTestProject) " />
  <Import Project="eng\targets\FunctionalTestWithAssets.props" Condition=" $(IsTestProject) " />
```

To be consistent with how the .targets files are imported.